packages/async-apollo-provider/src/schemaLoader.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import fp from 'lodash/fp';
+
+export const getFragmentsSchema = async (uri: string): ?Object => {
+  const result = await fetch(uri, {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({
+      query: `
+        {
+          __schema {
+            types {
+              kind
+              name
+              possibleTypes {
+                name
+              }
+            }
+          }
+        }
+      `,
+    }),
+  }).then(result => result.json());
+
+  if (fp.isNil(result.data)) {
+    return null;
+  }
+  else {
```

Comment:
You can write the following logic without `else` block.

Reply:
BTW, why didn't eslint catch this case? Maybe we should add rule? 
@ej9x @zouxuoz

Reply:
@vorobeez Do you mean the `no-else-return` rule?

Reply:
yep

Reply:
@vorobeez I think `else` is unnecessary here.

Reply:
and 

```js
}
else {
```

should be 

```js
} else {
```